tasks/unit-tests/testwasher_tests.py

```diff
@@ -30,3 +30,148 @@ def test_should_fail_failing_tests(self):
         module_path = "tasks/unit-tests/testdata"
         non_flaky_failing_tests = test_washer_3.get_non_flaky_failing_tests(module_path)
         self.assertEqual(non_flaky_failing_tests, {"github.com/DataDog/datadog-agent/pkg/gohai": {"TestGetPayload"}})
+
+    def test_should_mark_parent_flaky(self):
+        test_washer = TestWasher(
+            test_output_json_file="test_output_failure_parent.json",
+            flakes_file_path="tasks/unit-tests/testdata/flakes_2.yaml",
+        )
+        module_path = "tasks/unit-tests/testdata"
+        non_flaky_failing_tests = test_washer.get_non_flaky_failing_tests(module_path)
+        print(non_flaky_failing_tests)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** Did you forget to remove this print ?‏